src/asmcnc/apps/maintenance_app/widget_maintenance_z_lubrication_reminder.py

```diff
@@ -145,6 +145,8 @@ def update_strings(self):
         self.time_since_srew_lubricated_label.text = self.l.get_bold("TIME SINCE LEAD SCREW LUBRICATED")
         self.hours_label = self.l.get_str("hours")
 
+        self.update_label_font_size(self.time_since_srew_lubricated_label)
```

Comment:
just noticed the typo on this (srew). definitely my fault. if you get the chance to fix it (and it's related references) then please do, but it's not mission critical.